src/main/java/frc/robot/commands/AlignToPose.java

```diff
@@ -10,7 +10,7 @@
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.generated.TunerConstants;
```

Comment:
The only place that should import the generated TunerConstants should be the place we create the drivetrain.  The remaining places should get the values from the Drivetrain somehow